src/qml/NightstandWatchfacePage.qml

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2022 - Timo Könnecke <github.com/eLtMosen>
+ *               2022 - Darrel Griët <dgriet@gmail.com>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.12
+import Qt.labs.folderlistmodel 2.1
+import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
+import Nemo.Configuration 1.0
+import Nemo.Time 1.0
+
+Item {
+
+    property alias displayAmbient: compositor.displayAmbient
+    property string assetPath: "file:///usr/share/asteroid-launcher/"
+
+    ConfigurationValue {
+        id: watchfaceNightstandSource
+
+        key: "/desktop/asteroid/nightstand/watchface"
+        defaultValue: assetPath + "watchfaces/005-analog-nordic.qml"
```

Comment:
Any particular reason why you used this watchface instead of the default 000 one?
(I'm guessing just to differentiate, but number 005 feels a bit random :wink: )

Reply:
Ideally, I'd have used a stock watchface suitable for this mode, but none exist.  Number 005 was specifically chosen because it looks different from the default.  Otherwise the user would have no visual indication that anything was different.